### PR TITLE
Removes status checks from multiple repositories

### PR DESCRIPTION
**Pull request by PowerSchill**
## Description

Defaults to org default for status checks

### Issues Resolved

[List any existing issues this PR resolves]

### Check List
- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by Xorima** [comment]
Do not merge currently,state file is broken, fixing!

File: language-chef.tf [hunk 1]
```diff
@@ -2,7 +2,6 @@ module "language-chef" {
   source        = "./modules/repository"
   name          = "language-chef"
   cookbook_team = github_team.language-chef.id
-  status_checks = ["ci/circleci: lint"]
```
Comment (Xorima) [inline language-chef.tf]: This does not run the same lint as normal usecase, so may be wrong?

Comment (PowerSchill) [reply language-chef.tf]: I wasn't sure but I thought we could just add a circleci config to run whatever tests we want as long as final was the last one.

Comment (Xorima) [reply language-chef.tf]: We can do, just wanted to highlight this :)

**Pull request merged**